### ISSUE TITLE
Как пользоваться?

### ISSUE THREAD

**Issue by Jorah-M** [t1m0n/air-datepicker#443]
Простите за глупый и довольно общий вопрос, но я пока не слишком опытен, и мне не очень понятно, как пользоваться air-datepicker без jquery. В документации написано, что это возможно, но по примерам я не очень понял, как применять либу, допустим, в реакте. Поискав подобные вопросы в issues, я нашел лишь старые ответы, где используется jquery. 
Можете ли вы для таких тугодумов, как я, расписать реальный пример применения в функциональном компоненте реакта?

**Comment by t1m0n** [comment]
Добрый день! А что конкретно не понятно? В документации приведены примеры, как подключить и как инициализировать календарь. Что касается React.js, то есть [ответ на StackOverfow](https://stackoverflow.com/questions/44731221/how-integrate-jquery-plugin-in-react) - там используется старая версия, но для новой принцип использования точно такой же.

Если вам необходимо использовать в функциональном компоненте, то инициализация будет происходить в `useEffect` а не в `componentDidMount`

**Comment by t1m0n** [comment]
Да, все верно, там используется jQuery. Примеры без jQuery есть на [странице документации](https://air-datepicker.com/ru/examples), как я уже и говорил. Вы задаете базовые вопросы, которые не имеют отношения к календарю. Почитайте по-больше про основы JS, про классы, прототипы, работу с DOM, а уже потом беритесь за React - так будет гораздо проще  разобраться и в реакте, и других библиотеках.

Сделал для вас пример с функциональным компонентом - https://codesandbox.io/s/air-datepicker-react-example-mskzx?file=/src/datepicker-react.js
